config-templates/processFilterConfigurations-example.md

```diff
@@ -145,3 +145,16 @@ Example how to enable filter ForceAup:
 
 3.Fill the attributes and set list of required Aups (attr reqAups) and voShortNames (optional) for each facility
 
+## AttributeMap
+
+Example how to enable filter AttributeMap:
```

Comment:
```suggestion
This filter maps attribute names according to a service specific map from Perun. It can be used to achieve compatibility with a SP which requires specific non-standard attribute names.

Example how to enable filter AttributeMap:
```